test/attr/attr_abi.swift

```diff
@@ -1,7 +1,21 @@
-// RUN: %target-typecheck-verify-swift -enable-experimental-feature Extern -enable-experimental-feature ABIAttribute -parse-as-library
+// RUN: %target-typecheck-verify-swift -enable-experimental-feature Extern -enable-experimental-feature ABIAttribute -enable-experimental-feature AddressableParameters -enable-experimental-feature NoImplicitCopy -enable-experimental-feature SymbolLinkageMarkers -enable-experimental-feature StrictMemorySafety -enable-experimental-feature LifetimeDependence -enable-experimental-feature CImplementation -enable-experimental-feature ExecutionAttribute -import-bridging-header %S/Inputs/attr_abi.h -parse-as-library -Rabi-inference -debugger-support
 
 // REQUIRES: swift_feature_ABIAttribute
+// REQUIRES: swift_feature_AddressableParameters
+// REQUIRES: swift_feature_CImplementation
+// REQUIRES: swift_feature_ExecutionAttribute
 // REQUIRES: swift_feature_Extern
+// REQUIRES: swift_feature_LifetimeDependence
+// REQUIRES: swift_feature_NoImplicitCopy
+// REQUIRES: swift_feature_StrictMemorySafety
+// REQUIRES: swift_feature_SymbolLinkageMarkers
+
+import _Differentiation
```

Comment:
Missing a `REQUIRES: differentiable_programming` if one wants to import `_Differentiation`.